middle_end/flambda2/from_lambda/closure_conversion.ml

```diff
@@ -2366,7 +2384,15 @@ let close_functions acc external_env ~current_region function_declarations =
               symbol = None
             }
         in
-        Function_slot.Map.add function_slot approx approx_map)
+        let approx_map =
+          Function_slot.Map.add function_slot approx approx_map
+        in
+        match Function_decl.calling_convention decl with
+        | Normal_calling_convention -> approx_map
+        | Unboxed_calling_convention (_, _, unboxed_function_slot) ->
+          (* CR mshinwell: add a proper approximation here *)
```

Comment:
If we don't have a proper approximation, we won't be able to inline the stubs, so unboxing is never going to be beneficial (in classic mode).

Reply:
Unboxing is now disabled in classic mode. If/when we decide to enable it, we'll need to revisit this CR.